### PR TITLE
388-cookies

### PR DESCRIPTION
**Pull request by ikovalyov**

### CODE REVIEW

**Comment by ikovalyov** [comment]
[Cookies](https://trello.com/c/7bnObeLm/388-cookies)

File: src/Model/DisabledSessionManager.php [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+
+use Zend\Cache\Storage\StorageInterface;
+use Zend\EventManager\EventManagerInterface;
+use Zend\ServiceManager\ConfigInterface;
+use Zend\Session\Exception\RuntimeException;
+use Zend\Session\SaveHandler\SaveHandlerInterface;
+use Zend\Session\SessionManager;
+
+/**
+ * Class StorageModel
```
Comment (Xapp73) [inline src/Model/DisabledSessionManager.php]: DisabledSessionManager

File: src/Model/DisabledSessionManager.php [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+
+    /**
+     * @var array
+     */
+    private $data;
+
+    /**
+     * Constructor
+     *
+     * @param  ConfigInterface|null           $config
```
Comment (Xapp73) [inline src/Model/DisabledSessionManager.php]: strange positioning

File: src/Model/DisabledSessionManager.php [hunk 3]
```diff
@@ -0,0 +1,181 @@
...
+    public function destroy(array $options = null)
+    {
+        return;
+    }
+
+    /**
+     * @param string $name
+     * @return DisabledSessionManager
+     */
+    public function setName($name)
```
Comment (Xapp73) [inline src/Model/DisabledSessionManager.php:81]: type (and further)

**Review by Xapp73 — APPROVED** [review]

File: src/Model/DisabledSessionManager.php (hunk 3 above)
Comment (ikovalyov) [reply src/Model/DisabledSessionManager.php:81]: It comes from interface `public function setName($name);` declaration

**Review by Xapp73 — APPROVED** [review]

**Pull request merged**